go.mod

```diff
@@ -5,7 +5,7 @@ go 1.23.2
 require (
 	github.com/AlessandroPomponio/go-gibberish v0.0.0-20191004143433-a2d4156f0396
 	github.com/caarlos0/env/v9 v9.0.0
-	github.com/cilium/ebpf v0.17.3
+	github.com/cilium/ebpf v0.17.2
```

Comment:
Any reason to downgrade cilium ebpf, or it's an accident?

Reply:
Sorry, this was just to test the GH action